files/templates/alertmanager/alertmanager.yaml

```diff
@@ -617,6 +617,11 @@ inhibit_rules:
   - cancel_if_cluster_control_plane_unhealthy=true
   equal: [cluster_id]
 
+- source_matchers:
+  - cluster_control_plane_unhealthy=true
+  target_matchers:
+  - cancel_if_any_cluster_control_plane_unhealthy=true
```

Comment:
Maybe you don't need the any section?

Reply:
There were [a couple of sloth-based alerts](https://github.com/giantswarm/sloth-rules/pull/181/files) and [one prometheus-rules alert](https://github.com/giantswarm/prometheus-rules/pull/1167/files#diff-a30358e5d9e631784b9e7bdfbb678e0d12fd23d152089b531231ed66041872fc) using the "any" version of the apiserver_down inhibition so I've done one also for this to maintain parity

Reply:
Right but if it's thé same inhibition in the end maybe we could have only 1 ?

Reply:
Oh there's one for WC and MCs? Then that's fine. I m wondering if we need thé différence but having it will not break anything 😀